build.gradle

```diff
@@ -1,7 +1,12 @@
 // BEGIN ALL PROJECTS //
 allprojects {
     apply plugin: 'java'
+    apply plugin: 'project-report' // todo delete me
 }
+
+// for some reason this does not work if applied in allprojects.
+apply plugin: 'distribution'
```

Comment:
Can you move this to the `// ROOT PROJECT` section?

Reply:
Actually, do we even need this? The distribution plugin is applied specifically to the kcbq-confluent package down below.